projects/Dispersion/Dispersion.cpp

```diff
@@ -106,6 +106,10 @@ namespace projects {
       if(hook::END_OF_TIME_STEP == stage) {
          int myRank;
          MPI_Comm_rank(MPI_COMM_WORLD,&myRank);
+
+         cerr << "Warning, this function does virtually nothing, since B and E are not updated in DCCRG cell params since May 2019" << endl;
+         cerr << "If this is undersirable to you, please implement writing the fields out of fsgrid objects" << endl;
+
          vector<Real> localRhom(P::xcells_ini, 0.0),
                      outputRhom(P::xcells_ini, 0.0),
                      localPerBx(P::xcells_ini, 0.0),
```

Comment:
These declarations can be deleted too.

Reply:
I would rather let it reduce and write 0 than some uninitialized values

Reply:
But we don't want any writing of E And B for now. Just remove all mentions of perB? and E?, only leave a working output of rhom and whoever picks up at a later time can take inspiration and write equivalent fsgrid code for E or B.

Reply:
Ok